connect/runtime/src/test/java/org/apache/kafka/connect/integration/BlockingConnectorTest.java

```diff
@@ -527,7 +527,7 @@ public void maybeBlockOn(String block) {
                     }
                 }
             } else {
-                log.debug("Will not block on {}", block);
+                log.trace("Will not block on {}", block);
```

Comment:
I changed it to trace because there's a lot of noise in IT's from the poll() method.